zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Notebook.java

```diff
@@ -1020,6 +1020,9 @@ public void removeCron(String id) {
     } catch (SchedulerException e) {
       logger.error("Can't remove quertz " + id, e);
     }
+    Note notebook = getNote(id);
+    Map<String, Object> config = notebook.getConfig();
+    config.put("cron", null);
```

Comment:
why is the note still getting run if `quartzSched.deleteJob` is called?

Reply:
@felixcheung 
When zeppelin is restarting it bypass all notes and look at them's cron config. So note again become cronable.

Reply:
when the note is deleted it should go to trash, right? is there something I've missed here? it's still found when zeppelin starts again?

Reply:
@felixcheung 
The difference with trashed note is only `~Trash/` prefix. So zeppelin can found it.